front/app/containers/Admin/tools/PowerBI/PowerBITemplates/messages.ts

```diff
@@ -50,7 +50,8 @@ export default defineMessages({
   },
   supportLinkDescription: {
     id: 'app.containers.Admin.tools.powerBITemplates.supportLinkDescription',
-    defaultMessage: 'Further details about using your CitizenLab data in Power BI can be found in our {link}.',
+    defaultMessage:
+      'Further details about using your CitizenLab data in Power BI can be found in our {link}.',
   },
```

Comment:
Not sure where these PowerBI changes are coming from? You didn't make these changes right?